src/hotspot/share/gc/g1/c2/g1BarrierSetC2.cpp

```diff
@@ -248,9 +248,9 @@ void G1BarrierSetC2::pre_barrier(GraphKit* kit,
       pre_val = __ load(__ ctrl(), adr, val_type, bt, alias_idx, false, MemNode::unordered, LoadNode::Pinned);
     }
 
-    // if (pre_val != NULL)
+    // if (pre_val != null)
```

Comment:
```suggestion
    // if (pre_val != nullptr)
```
This is converted wrongly in a comment